homeassistant/components/sma/sensor.py

```diff
@@ -211,6 +215,7 @@ def device_info(self) -> DeviceInfo | None:
             model=self._device_info["type"],
             name=self._device_info["name"],
             sw_version=self._device_info["sw_version"],
+            configuration_url=self._device_info["configuration_url"],
```

Comment:
Import constant from core `const.py`: https://github.com/home-assistant/core/blob/9daf2ee65d617055b7b17783a369e4b5f5fe4006/homeassistant/const.py#L355

```suggestion
            configuration_url=self._device_info[ATTR_CONFIGURATION_URL],
```